packages/synthetix-governance/tasks/fast-forward.js

```diff
@@ -0,0 +1,31 @@
+const { task } = require('hardhat/config');
+const { fastForward } = require('@synthetixio/core-js/utils/hardhat/rpc');
+
+const units = {
+  seconds: 1,
+  minutes: 60,
+  hours: 60 * 60,
+  days: 60 * 60 * 24,
+  weeks: 60 * 60 * 24 * 7,
+};
```

Comment:
Maybe this conversion tool can go to `core-js/misc/dates` ?

Reply:
So, it would be like `parseTime('3 hours') // => 3 * 60 * 60`

Reply:
Hmm... tbh this kind of tooling is overkill. It's more explicit to think about the problem than to remember where the tool is. So I think it's good here. But as you prefer.